test/functional/cmdLineTests/ignoreUnrecognizedVMOptions/playlist.xml

```diff
@@ -31,7 +31,22 @@
 		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) -DTESTDIR=$(Q)$(TEST_RESROOT)$(Q) -DRESJAR=$(CMDLINETESTER_RESJAR) -DEXE=$(SQ)$(JAVA_COMMAND)$(SQ) -jar $(CMDLINETESTER_JAR) -config $(Q)$(TEST_RESROOT)$(D)ignoreunrecognizedvmoptions.xml$(Q) -verbose -explainExcludes -nonZeroExitWhenError; \
 	$(TEST_STATUS)</command>
 		<levels>
-			<level>sanity</level>
+			<level>extended</level>
```

Comment:
FYI, I asked for this change, to move this testing to extended. I don't think it needs to be in sanity, as the implementation would be rarely changed and unlikely to break normally.